### PR TITLE
VEOSS-1810 | Build Self Exclude No Longer Selling Goods

### PR DESCRIPTION
**Pull request by Lee-Powell**

### CODE REVIEW

File: .g8/checkboxPage/app/controllers/$className$Controller.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import pages.$className$Page
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.$className$View
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class $className$Controller @Inject()(
```
Comment (ZDevelop94) [inline .g8/checkboxPage/app/controllers/$className$Controller.scala:17]: Still a newbie and but it appears this allows multiple routes to use the same controller. Wouldn't it be more maintainable to have explicit named controllers? So it is easier to maintain and not so coupled together? I might be wrong here so don't be harsh :)

File: .g8/checkboxPage/app/pages/$className$Page.scala [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package pages
+
+import models.$className$
+import play.api.libs.json.JsPath
+
+case object $className$Page extends QuestionPage[Set[$className$]] {
```
Comment (ZDevelop94) [inline .g8/checkboxPage/app/pages/$className$Page.scala:6]: Yeah I am not to sure about `$className$Page` it might make the journeys very hard to follow.

File: .g8/checkboxPage/app/controllers/$className$Controller.scala (hunk 1 above)
Comment (ZDevelop94) [reply .g8/checkboxPage/app/controllers/$className$Controller.scala:17]: Arrr I see it is from the generator ignore me

**Review by ZDevelop94 — APPROVED** [review]

**Review by DevAndyroo — APPROVED** [review]
